data/language/en-GB.txt

```diff
@@ -4121,6 +4121,7 @@ STR_5811    :{SMALLFONT}{BLACK}If checked, you can have up to{NEWLINE}255 cars p
 STR_5812    :Show multiplayer window
 STR_5813    :{OPENQUOTES}{STRING}{ENDQUOTES}
 STR_5814    :{WINDOW_COLOUR_1}{OPENQUOTES}{STRING}{ENDQUOTES}
+STR_5871    :Krone (kr)
```

Comment:
I think that this should be moved until after STR_5870, to maintain order (and prevent confusion).